### ISSUE TITLE
Can't get the latest newpipe release installed, get install error through an update and through a fresh install

### ISSUE THREAD

**Issue by dimyself** [TeamNewPipe/NewPipe#10318]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

v0.25.2

### Steps to reproduce the bug

i can't get the latest version to install. i'm on current version 0.25.1, and when i try to install the apk using the package manager (from github releases), i get error: "app not installed as package conflicts with an existing package"

i also try to install the the same release and github source, but installing it through obtainium, it gives me more info to the error: "conflict: org.schabi.newpipe". 

i have tried updating my current version, and i have also tried uninstalling newpipe and installing 0.25.2 as a fresh install. i get the same error no matter what

i'm not sure how to fix this as i don't have root access on my phone to access the apps directly and remove the old directory structure?

i've also tried clearing cache, storage, and disabling the app and then trying the install... same error

i'm using GrapheneOS on a Pixel 6 pro

thank you! 

### Affected Android/Custom ROM version

Android 13 / GrapheneOS

### Affected device model

Pixel 6 Pro

**Comment by andreapx** [comment]
I'm having this problem too, and NewPipe everyonce in a while (while not using it) throw an _ui error_

**Comment by dimyself** [comment]
thanks for the response @SameenAhnaf  .... i did initially install using Fdroid

since i uninstalled newpipe and tried installing it through the github source... now it wont let me install using Fdroid either: "failed to install due to an unknown error"

newpipe wont install at all now... you mentioned the keys are different and thats why they interfere with each other... is there a way i can clear the keys to i can install newpipe again? 

or maybe you know of another way i can fix the install errors?

thank you

**Comment by ChiefMikeK** [comment]
:shipit: If you are going to update via f-droid and have installed it from github then add this REPO to your f-droid app 
---
https://newpipe.net/FAQ/tutorials/install-add-fdroid-repo/  
or uninstall it each time upon changing repo-source.

**Comment by dimyself** [comment]
yes, i did a little digging around... i'm on grapheneOS which i setup with
different app profiles

idk why, but since i have newpipe installed in 2 different profiles for
some reason they clash... maybe they share the install directory or
something

anyway, after removing my other profile newpipe install, i had no issues :/

sorry about that!! i didnt see the other install, and it didnt occur to me
it would interfere since its separate.

this can be closed from my end! i appreciate everyones help with this!!
